### PR TITLE
Add volk_32fc_accumulator_s32fc

### PR DESCRIPTION
**Pull request by Ka-zam**
Adding accumulator for complex vectors.

~/src/volk$ ~/local/bin/volk_profile -n -R 32fc_accu

Warning: this IS a dry-run. Config will not be written!
RUN_VOLK_TESTS: volk_32fc_accumulator_s32fc(131071,1987)
u_avx completed in 63.012 ms
u_sse completed in 122.686 ms
generic completed in 245.437 ms
Best aligned arch: u_avx
Best unaligned arch: u_avx
Warning: this was a dry-run. Config not generated

### CODE REVIEW

File: kernels/volk/volk_32fc_accumulator_s32fc.h [hunk 1]
```diff
@@ -0,0 +1,159 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2012, 2019 Free Software Foundation, Inc.
```
Comment (michaelld) [inline kernels/volk/volk_32fc_accumulator_s32fc.h]: Copyright is just 2019; please remove the 2012.

File: kernels/volk/volk_32fc_accumulator_s32fc.h [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GNU Radio; see the file COPYING.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street,
+ * Boston, MA 02110-1301, USA.
+ */
+
+/*!
+ * \page volk_32fc_accumulator_s32fc
```
Comment (michaelld) [inline kernels/volk/volk_32fc_accumulator_s32fc.h:24]: Volk kernels are traditionally named with the action being taken, not the object taking the action. So, not "accumulator", but rather "accumulate". Hence, please change the filename and all internal references to "accumulate".

**Review by michaelld — CHANGES_REQUESTED** [review]
A couple minor changes, then it'll ready to go IMHO. Thank you for this kernel!

**Comment by michaelld** [comment]
@Ka-zam do you have a CLA (code license assignment) in place? Without it, we can't merge this much code. If you don't have one and you want one, send an email to Ben Hilburn <bhilburn@gnuradio.org> to get the conversation started; feel free to note the URL for this PR.

File: kernels/volk/volk_32fc_accumulator_s32fc.h (hunk 2 above)
Comment (Ka-zam) [reply kernels/volk/volk_32fc_accumulator_s32fc.h:24]: Well, there's already this so tradition has failed us:
[http://libvolk.org/doxygen/volk_32f_accumulator_s32f.html](url)

:smile:

Do you want:
1. I change to accumulate and now we have both "accumulate" and "accumulator" depending on type.
2. Leave like it is, i.e. "accumulator" for consistency.
3. Make another PR changing volk_32f_accumulator_s32f to use "accumulate".
4. Have both names point to the same function, i.e both "accumulate" and "accumulator" for both types.

**Comment by Ka-zam** [comment]
I'll get the CLA process started.

File: kernels/volk/volk_32fc_accumulator_s32fc.h (hunk 2 above)
Comment (michaelld) [reply kernels/volk/volk_32fc_accumulator_s32fc.h:24]: well how about that! I somehow didn't see that kernel in my search ... you are totally correct! Let's just leave this one as "accumulator" to match the other one. That way there's no API change from past, just augmenting it. Resolving this request!

**Comment by michaelld** [comment]
Ci failures have nothing to do with this PR. Leaving the kernel name alone since there's already another of similar name. LGTM ... will now wait for the CLA to come through.

**Comment by Ka-zam** [comment]
Added NEON version tested on Aarch32 and Aarch64

`pi@raspberrypi:~/src/kazam/volk/build $ ~/local/bin/volk_profile -R accu -v 2048 -i 100000

RUN_VOLK_TESTS: volk_32f_accumulator_s32f(2048,100000)
no architectures to test
RUN_VOLK_TESTS: volk_32fc_accumulator_s32fc(2048,100000)
neon_a16 completed in 351.853 ms
neon_a32 completed in 174.447 ms
neon_a64 completed in 155.109 ms
neon_a64pre completed in 167.476 ms
generic completed in 217.732 ms
Best aligned arch: neon_a64
Best unaligned arch: neon_a64

demouser@pi64 ~/src/kazam/volk/build $ ~/local/bin/volk_profile -n -R 32fc_accu -v 2048 -i 100000

Warning: this IS a dry-run. Config will not be written!
RUN_VOLK_TESTS: volk_32fc_accumulator_s32fc(2048,100000)
neon_a16 completed in 288.378 ms
neon_a32 completed in 127.246 ms
neon_a64 completed in 121.798 ms
neon_a64pre completed in 163.544 ms
generic completed in 225.768 ms
Best aligned arch: neon_a64
Best unaligned arch: neon_a64
Warning: this was a dry-run. Config not generated
`

**Comment by michaelld** [comment]
Nice to have neon on board too!

**Review by jdemel — APPROVED** [review]
LGTM

**Review by michaelld — APPROVED** [review]
LGTM ; just waiting for the CLA now ...

**Comment by jdemel** [comment]
@Ka-zam Where are we with the CLA now? 
@bhilburn do you know about this?

**Comment by jdemel** [comment]
@Ka-zam  Is there any update on the CLA?
We start to see merge conflicts and I'd be more than happy to avoid them by merging your PRs.

**Comment by jdemel** [comment]
@Ka-zam I assume there's still no update on your CLA? Same question as in #322 

Anyways, we adopted a new system: DCO. Thus, you only need to sign off your commits. `git commit -s` for new commits. The details are explained in `DCO.txt`. With `git commit -s` you add a `Signed off by ...` line to the end of your commit message. You can add

I got this approach to sign previous commits from [stackoverflow](https://stackoverflow.com/a/58599218)
```
git rebase --exec 'git commit --amend --no-edit -n -s' -i HEAD~8
```
I updated the option `-s` to lower case. Also, `HEAD~8` should refer to your 8 commits. It would be great if you could rebase your PR onto current master. In that case we could just merge it.

**Comment by jdemel** [comment]
@Ka-zam now that we could merge #457 would you be willing to rebase this PR as well?

**Comment by Ka-zam** [comment]
@jdemel Yes, of course.

**Pull request closed without merging**